eng/Versions.props

```diff
@@ -1,12 +1,12 @@
 <Project>
   <PropertyGroup>
     <!-- The .NET product branding version -->
-    <ProductVersion>5.0.0</ProductVersion>
+    <ProductVersion>6.0.0</ProductVersion>
     <!-- File version numbers -->
-    <MajorVersion>5</MajorVersion>
+    <MajorVersion>6</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
-    <PreReleaseVersionLabel>rc</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>alpha</PreReleaseVersionLabel>
```

Comment:
I'm curious why we are not starting with `preview 1`?

Reply:
I believe this is for ordering in the feeds as alpha will always be sorted as the oldest, so this gives us the flexibility to choose other prefixes before previews.

Reply:
Yep.